packaging/rhc-worker-script.spec

```diff
@@ -71,6 +71,17 @@ install -D -d -m 755 %{buildroot}%{rhc_worker_conf_dir}
 
 %changelog
 
+* Thu Aug 10 2023 Rodolfo Olivieri <rolivier@redhat.com> 0.3-1
+- Parse minimal yaml instead of raw bash script
+- Tidy up the modules and replace deprecated call WithInsecure
+- Add option to create config for rhc-worker-bash
+- Fix build for go1.16 by @r0x0d in #53
```

Comment:
I wouldn't link like this to PRs as it's difficult to see it locally when checking a package changelog. I'd just go with the summary and remove the `by person` as well

Reply:
Oh, that's true! Forgot that one.

Reply:
done